sim/shaman/fire_elemental_pet.go

```diff
@@ -32,17 +32,21 @@ func (shaman *Shaman) NewFireElemental(bonusSpellPower float64) *FireElemental {
 	}
 	fireElemental.EnableManaBar()
 	fireElemental.AddStatDependency(stats.Intellect, stats.SpellPower, 1.0)
+	fireElemental.AddStatDependency(stats.Intellect, stats.AttackPower, 1.0)
 	fireElemental.AddStat(stats.SpellPower, -10)
+	fireElemental.AddStat(stats.AttackPower, -10)
 	fireElemental.EnableAutoAttacks(fireElemental, core.AutoAttackOptions{
 		MainHand: core.Weapon{
-			BaseDamageMin:  429, //Estimated from beta testing
-			BaseDamageMax:  463, //Estimated from beta testing
-			SwingSpeed:     2,
-			CritMultiplier: 2.66, //Estimated from beta testing
-			SpellSchool:    core.SpellSchoolFire,
+			BaseDamageMin:     429, //Estimated from beta testing
+			BaseDamageMax:     463, //Estimated from beta testing
+			SwingSpeed:        2,
+			CritMultiplier:    2.66, //Estimated from beta testing
+			AttackPowerPerDPS: 2,
```

Comment:
Do you have a reason to think AttackPowerPerDPS isn't 14? If not I'd rather see it left at default and scale inherited AP up by 7, since AP/14 is standard across all other player-controlled things

Reply:
i can do that